src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -515,6 +515,46 @@ public string[] ParameterSetName
         private string[] _parameterSetName;
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Type accelerators.BuiltIn Accelerators.Should have all the type accelerators
```powershell
Expected 104, but got 105.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\TypeAccelerator.Tests.ps1: line 441
441:             $TypeAccelerators.Count | Should -Be $totalAccelerators
```